java/ql/src/Telemetry/AutomodelFrameworkModeCharacteristics.qll

```diff
@@ -167,14 +167,15 @@ class FrameworkModeMetadataExtractor extends MetadataExtractor {
 
   override predicate hasMetadata(
     Endpoint e, string package, string type, boolean subtypes, string name, string signature,
-    int input
+    int input, string parameterName
   ) {
     exists(Callable callable |
       e.asParameter() = callable.getParameter(input) and
       package = callable.getDeclaringType().getPackage().getName() and
       type = callable.getDeclaringType().getErasure().(RefType).nestedName() and
       subtypes = this.considerSubtypes(callable) and
-      name = e.toString() and
+      name = callable.getName() and
+      parameterName = e.toString() and
```

Comment:
```suggestion
      parameterName = e.asParameter().getName() and
```
I don't know for sure that this is the right way to express this, but we should be using something like this instead of `toString`.

Reply:
OMG, I looked for this but didn't find it. Thx!